.travis.yml

```diff
@@ -11,7 +11,6 @@ env:
     - MOLECULE_DISTRO: centos6
     - MOLECULE_DISTRO: ubuntu1804
     - MOLECULE_DISTRO: ubuntu1604
```

Comment:
why are we not adding centos8 here as a distro?

Reply:
Oh, I didn't assume geerlingguy has made a Docker image for CentOS 8, but he actually has. I will add this.